### DISCUSSION TITLE
Login page doesn't respect Application-URL

### DISCUSSION THREAD

**Discussion by Floyer007** [kanboard/kanboard#5193] [Installation & Configuration]
### Actual behaviour

While using kanboard into a subfolder with a "Application-URL" (e.g. https://url.tld/kanboard), a successful login (via https://url.tld/kanboard/?controller=AuthController&action=login) redirects to a wrong url (https://url.tld/).
To use kanboard the url (https://url.tld/kanboard) has to be opened manually again.

### Expected behaviour

After a successful login the url should direct to the defined "Application-URL" (https://url.tld/kanboard) and the kanboard is visible.

### Steps to reproduce

- Setup a subfolder usage
- Login to your kanboard

### Configuration

- Kanboard version:  v1.2.14 
- Database type and version:  sqlite
- PHP version:  7.3.16
- OS: Debian 10 (buster)
- Browser: Firefox/77.0

**Comment by cjohnsonuk** [comment]
In settings 
in application settings
there is a value called application URL
Make sure its set and includes your subfolder.  Mine is in a subfolder and this setting has worked on at least the last 3 versions including 1.2.14
